scripts/release.sh

```diff
@@ -60,8 +60,18 @@ if [[ $(git status -uno --porcelain) ]]; then
   Die 'There are uncommitted changes.'
 fi
 
-# Make sure client is up to date with the latest changes.
-git checkout master
+if [[ "${SUFFIX}" = "lts" && "${VERSION}" != *.*.0 ]]; then
```

Comment:
So if you run `./release.sh lts 5.1.3`, you will take the `else` branch, and `BASE_BRANCH` will be `master`. Is this correct?

Reply:
Also update the usage on line 2.
```
#!/bin/bash -
# Usage: ./release.sh <dependencies|bom> <release version>
```

Reply:
> if you run ./release.sh lts 5.1.3, you will take the else branch, and BASE_BRANCH will be master. Is this correct?

No, that command uses "lts-5.1.0" as BASE_BRANCH.

The second part of the if-condition is 'not matching'.

Reply:
The line 2 is updated.